modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -112,6 +112,15 @@ Number of unavailable partitions (the partitions that lack quorum among their re
 
 ---
 
+[[redpanda_cluster_latest_cluster_metadata_manifest_age]]
+=== redpanda_cluster_latest_cluster_metadata_manifest_age
+
+The age in seconds since the last time Redpanda uploaded metadata files to tiered storage for your cluster. A value of `0` indicates metadata has not yet been uploaded. When performing a xref:manage:whole-cluster-restore.adoc[whole cluster restore] operation, the metadata for the new cluster will not have any changes made to the source cluster which are newer than this age.
```

Comment:
```suggestion
The amount of time in seconds since the last time Redpanda uploaded metadata files to Tiered Storage for your cluster. A value of `0` indicates metadata has not yet been uploaded. When performing a xref:manage:whole-cluster-restore.adoc[whole cluster restore] operation, the metadata for the new cluster will not have any changes made to a source cluster that is newer than this age.
```